client/app/bundles/shared/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import css from './Pagination.scss';
+
+export default class Pagination extends React.Component {
+
+   constructor(props) {
```

Comment:
Hi @nickeyvee . Indentation needs to be 2 spaces here, not 3. That's one of the reasons the Circle code integration is failing. Other errors can be seen here: https://circleci.com/gh/ifmeorg/ifme/2393?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link